modules/300-prometheus/images/grafana-v10/grafana_home_dashboard.json

```diff
@@ -464,7 +464,7 @@
         {
           "editorMode": "code",
           "exemplar": true,
-          "expr": "count by (controller_type) (kube_controller_pod)",
+          "expr": "count by (controller_type) (count by (controller_type, controller_name) (kube_controller_pod))",
```

Comment:
Hi! Thanks for the contribution.

The current PromQL might return incorrect results if there are controllers with the same name but in different namespaces. To fix this, please update the query like this:

```
count by (controller_type) (count by (controller_type, controller_name, namespace) (kube_controller_pod))
```

This ensures names are grouped by namespace to avoid collisions.

Thanks!